doc/tutorial/read_mapper/read_mapper_step2.cpp

```diff
@@ -51,11 +51,7 @@ void map_reads(std::filesystem::path const & query_path,
                                                     seqan3::search_cfg::error_count{errors}} |
                                                 seqan3::search_cfg::hit_all_best{};
 
-#if SEQAN3_WORKAROUND_GCC_93983
-    for (auto && record : query_file_in /*| seqan3::views::take(20)*/)
-#else // ^^^ workaround / no workaround vvv
     for (auto && record : query_file_in | seqan3::views::take(20))
```

Comment:
Simple, because

https://github.com/seqan/seqan3/blob/4f36e068f2b00029de5768b2b6cf1c279567c37e/include/seqan3/io/detail/take_view.hpp#L619

isn't deprecated xD

Reply:
Ah, yes I remember :upside_down_face: